src/model.jl

```diff
@@ -84,16 +84,16 @@ Base.parent(m::AbstractModel) = getfield(m, :parent)
 setparent(m::AbstractModel, newparent) = @set m.parent = newparent
 
 params(m::AbstractModel) = params(parent(m))
-stripparams(m::AbstractModel) = stripparams(parent(m))
+strip(m::AbstractModel) = strip(parent(m))
```

Comment:
As discussed in the main thread, I would suggest not doing this.

Reply:
Yeah, we just need a good verb to replace it